src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/GivenIAmConfirmingMyIdentity/WhenViewingConfirmMyIdentityPage.cs

```diff
@@ -16,20 +17,45 @@ public class WhenViewingConfirmMyIdentityPage
         [Test, PageAutoData]
         public async Task The_page_shows_my_email_address(
             [Frozen] Mock<IApiClient> api,
-            [Frozen] ClaimsPrincipal user,
             ConfirmYourIdentityModel sut,
+            ClaimsPrincipal user,
             Registration registration)
         {
             user.AddIdentity(new ClaimsIdentity(new[]
             {
-                new Claim("sub", registration.Id.ToString())
+                new Claim("registration_id", registration.Id.ToString())
             }));
             registration.UserId = null;
             api.Setup(x => x.GetRegistration(registration.Id)).Returns(Task.FromResult(registration));
 
-            await sut.OnGetAsync();
+            await sut.OnGetAsync(new RegistrationUser(user));
```

Comment:
This name RegistrationUser looks wrong. 

Perhaps RegisteredUser, is a better name